tests/testProvenanceSetup.py

```diff
@@ -132,15 +136,15 @@ def testAddCmd(self):
         self.assertEquals(cmds[0][0], "recProv.py")
         self.assertEquals(cmds[0][1], "-v")
         self.assertEquals(cmds[0][2], "4")
-        
+
         self.setup.addWorkflowRecordCmd("dbingest.py", "-u ray".split(),
                                         "/usr/local/bin/dbingest.py")
```

Comment:
A comment sounds wise. Unfortunately I don't understand Steve's explanation well enough to document what is going on. (Is the code is actually putting a file in /usr/local/bin or just telling provenance to record that a file exists in /usr/local/bin?). I'm not complaining about the code, just saying I'm not too ignorant to explain it.